dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -980,16 +980,43 @@
 	</Target>
 
 	<Target Name="_FindAotCompiler" DependsOnTargets="_ComputeVariables">
+		<PropertyGroup>
+			<_AotCompilerCachePath>$(DeviceSpecificIntermediateOutputPath)aot-compiler-path-$(NETCoreSdkVersion).txt</_AotCompilerCachePath>
+		</PropertyGroup>
+
+		<!-- This task does not take a SessionId because we cache the path on Windows for remote builds to avoid a round-trip -->
+		<ReadLinesFromFile
+			Condition="Exists('$(_AotCompilerCachePath)')"
+			File="$(_AotCompilerCachePath)"
+		>
+			<Output TaskParameter="Lines" PropertyName="_AOTCompiler" />
```

Comment:
The only thing I would suggest, is `$(_AOTCompiler)` might collide with someone else trying to use the same name? Something in the .NET SDK or Mono workload.

In Android, I would name it `$(_AndroidAOTCompiler)` to avoid that, so I don't know if you normally use `Apple` as a prefix or something else.

Reply:
I'll do that in a different PR.

Reply:
https://github.com/xamarin/xamarin-macios/pull/17049